src/components/Icons/Icon.stories.tsx

```
@@ -0,0 +1,135 @@
+
+export const Confirm: Story = {
+  render: (args: React.ComponentProps<typeof IconConfirmCircle>) => (
+    <IconShowcase IconComponent={IconConfirmCircle} args={args} />
+  ),
+};
+
+export const Edit: Story = {
+  render: (args: React.ComponentProps<typeof IconEdit>) => <IconShowcase IconComponent={IconEdit} args={args} />,
+};
+
+export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)